lib/action.js

```diff
@@ -111,6 +111,31 @@ module.exports.actions = [
 		}
 	},
 
+	// Action to set an input field value
+	// E.g. "set field #username to example"
+	{
+		name: 'clear-field-value',
+		match: /^clear( field)? (.+?)$/i,
+		run: async (browser, page, options, matches) => {
+			const selector = matches[2];
+			try {
+				await page.evaluate(targetSelector => {
+					const target = document.querySelector(targetSelector);
+					if (!target) {
+						return Promise.reject(new Error('No element found'));
+					}
+					target.value = '';
+					target.dispatchEvent(new Event('input', {
```

Comment:
thinking between `input` and `change` events.

Reply:
You do want the `input` event. If you look at the [set-field-value](https://github.com/pa11y/pa11y/blob/1d31213b7403c09456a1f2f38aebef0182fd4552/lib/action.js#L95) action theres some extra bit that need to be performed to trigger any `setters` that have been added (such as in Reacts case).
The common functionality of these two actions could definitely be abstracted

Reply:
Thanks for the comment. I have added in this now